src/constraints/battery_degradation.jl

```diff
@@ -196,11 +254,15 @@ function add_degradation(m, p; b="ElectricStorage")
         )
         # add augmentation cost to objective
         # maintenance_cost_per_kwh must have length == length(days) - 1, i.e. starts on day 2
+
+        # create salvage value expression for objective, all 0s since we have no salvage value
+        s = zeros(length(months))
+        @expression(m, salv_value, sum(s[mth] for mth in months))
```

Comment:
why not just `@expression(m, salv_value, 0.0)`?

Reply:
I got rid of this expression. If we have to include salvage value in the objective, we can do it via `reopt.jl` file for replacement strategy only.